### DISCUSSION TITLE
Go Off Grid option for Tesla Powerwall

### DISCUSSION THREAD

**Discussion by naudecruywagen** [home-assistant/frontend#13780] [Other feature requests] [answered]
The Tesla app and the local web interface has an option to go off grid. 
There are times that I would like to go off grid and being able to schedule/automate it would be much better. 

Is it possible to add this option to the HA integration? (Probably as a switch)

Thanks!

![5757523D-6010-4A59-9B7B-8ACF9EDFDC7E](https://user-images.githubusercontent.com/1192894/190713907-7a9ccb8f-9466-4c75-baca-417f83ac6066.jpeg)

**Comment by ludeeus** [comment, marked as answer]
As this needs support in the integration this feature request is opened in the wrong place.
https://community.home-assistant.io/c/feature-requests/13

**Reply by naudecruywagen** [reply]
Ah thanks @ludeeus ! I didn’t know

**Reply by naudecruywagen** [reply]
Moved it https://community.home-assistant.io/c/feature-requests/13
